h2o-algos/src/main/java/hex/rulefit/Rule.java

```diff
@@ -150,6 +159,19 @@ public CategoricalThreshold(String[] catThreshold, int[] catThresholdNum) {
         return Math.abs(coefficient);
     }
 
+    byte[] ruleReducer(byte[][] bytes) {
+        byte[] ncs = MemoryManager.malloc1(bytes[0].length);
```

Comment:
move to RuleEnsamble

Reply:
would leave this here: when you have rule composed out of three conditions, you get three binary columns by evaluating these conditions on data and rule reducer reduces them to 1 column which represents this rule

Reply:
fixed